src/AxesUtil/AxesUtil.spec.js

```diff
@@ -80,4 +82,74 @@ describe('AxesUtil', () => {
     expect((typeof result.scale) === 'function').toEqual(true);
   });
 
+  it('can create d3 axis objects with custom format', () => {
+    const result = AxesUtil.createXAxis({
+      format: '04d'
+    }, scaleLinear());
+    expect((typeof result) === 'function').toEqual(true);
+    expect((typeof result.scale) === 'function').toEqual(true);
+  });
+
+  it('will fail to create a d3 axis object with broken format', () => {
+    /** lint needs doc */
+    const testfn = () => {
+      AxesUtil.createXAxis({
+        format: 'zzz'
+      }, scaleLinear());
+    };
+    expect(testfn).toThrow();
```

Comment:
```suggestion
    expect(() => {
      AxesUtil.createXAxis({
        format: 'zzz'
      }, scaleLinear());
    }).toThrow();
```

then you can remove the bogus jsdoc. Untested™

Reply:
Done in 27adb8a, thanks